internal/components/components.go

```diff
@@ -214,6 +215,7 @@ func Components() (component.Factories, error) {
 		awsxrayexporter.NewFactory(),
 		azuremonitorexporter.NewFactory(),
 		carbonexporter.NewFactory(),
+		coralogixexporter.NewFactory(),
```

Comment:
Adding the exporter to `internal/components` should be done on the last PR per the contribution guidelines. The main `go.mod` should have no changes either